<?xml version="1.0"?>

<launch>
<!--    <rosparam file="$(find stair_detection)/config/stair_config.yaml" command="load" /> -->

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find stair_complete)/config/offline_opt_visual.rviz" />

    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" />

    <node name="tf_world_set" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 1 world odom 10" />

    <node pkg="stair_complete" type="offline_opt_node" name="offline_opt" output="screen" launch-prefix="gnome-terminal -e"/>

    <param name="tsdf_save_path" value="$(find stair_complete)/saved_layer/big_truncation.vxblx" />

</launch>
